frontend/src/pages/HomePage.tsx

```diff
@@ -41,19 +45,19 @@ const HomePage = (): ReactElement => {
 
   const locationData: LocationCardData[] = [
```

Comment:
There was this `locationData` list defined already so that's reasonable to put the location-image mapping here. But  it's more straightforward logically to import the image and render them in `LocationCards`.  And from a performance point of view, importing unnecessary images high up in the ancestor and passing them as props both incur unnecessary latency. So consider switching this to `LocationCards`!